abdiff/core/collate_ab_transforms.py

```diff
@@ -218,39 +218,32 @@ def validate_output(dataset_path: str) -> None:
     and whether any or both 'record_a' or 'record_b' columns are
     totally empty.
     """
-    collated_parquet_glob = f"{dataset_path}/**/*.parquet"
-
     with duckdb.connect(":memory:") as con:
-        # check if the table is empty
-        record_count = con.execute(
-            "SELECT COUNT(*) FROM read_parquet($collated_parquet_glob)",
-            {"collated_parquet_glob": collated_parquet_glob},
-        ).fetchone()[
-            0
-        ]  # type: ignore[index]
+        # create view of collated table
+        con.execute(
+            f"""
+            CREATE VIEW collated AS (
+                SELECT * FROM read_parquet('{f"{dataset_path}/**/*.parquet"}')
```

Comment:
I think this will work given that we don't need any partitions from the dataset -- in fact, don't think there _are_ partitions -- but worth noting that if we _did_, we'd need to add `hive_partitioning=true`. 

Again, not needed here, just noting!